qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -155,6 +157,15 @@ def __init__(
             self._api_client = RuntimeClient(self._client_params)
             self._backend_allowed_list = self._discover_cloud_backends()
         else:
+            warnings.warn(
+                "The ibm_quantum channel option is deprecated will be sunset on July 1st. "
+                "After this date, ibm_cloud will be the only valid channel. "
+                "For information on migrating to the new IBM Quantum Platform on the "
+                "ibm_cloud channel, read our migration guide "
```

Comment:
```suggestion
                ""ibm_cloud" channel, review the migration guide "
```

Reply:
Is there a reason why this isn't a link?  Maybe you can't add links in this format?